shotgun_api3/shotgun.py

```diff
@@ -312,7 +314,7 @@ def __init__(self,
         self.config.convert_datetimes_to_utc = convert_datetimes_to_utc
         self.config.no_ssl_validation = NO_SSL_VALIDATION
         self._connection = None
-        self.__ca_certs = ca_certs
+        self.__ca_certs = ca_certs or os.environ.get('SHOTGUN_API_CACERTS')
```

Comment:
Would be nicer to structure this as

```python
if ca_certs is not None:
    self.__ca_certs = ca_certs
else:
   self.__ca_certs = os.environ.get('SHOTGUN_API_CACERTS')
```

That way if the passed in param for ca_certs just happens to have a False value the current behavior will be preserved.  Using that or construct can be more trouble than it is worth.